sgf-viewer/index.html

```diff
@@ -6,9 +6,10 @@
   <link rel="icon" type="image/svg+xml" href="/images/favicon.svg" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   <title>Adversarial Policies in Go - Game Viewer</title>
-  <script type="text/javascript" src="/wgo/wgo.min.js"></script>
-  <script type="text/javascript" src="/wgo/wgo.player.min.js"></script>
-  <link type="text/css" href="/wgo/wgo.player.css" rel="stylesheet" />
+  <script defer type="text/javascript" src="/wgo/wgo.min.js"></script>
+  <script defer type="text/javascript" src="/wgo/wgo.player.min.js"></script>
+  <link rel="preload" href="/wgo/wgo.player.min.css" as="style" onload="this.onload=null;this.rel='stylesheet'">
```

Comment:
add comment about why we defer load JS + CSS for wgo? and link https://web.dev/articles/defer-non-critical-css#optimize as a source for CSS deferral code?
but these comments will actually show up in the deployed website's source code so that's annoying